perllib/FixMyStreet/Roles/Cobrand/BulkyWaste.pm

```diff
@@ -589,7 +592,12 @@ sub bulky_nice_cancellation_cutoff_time {
 
 sub _bulky_cancellation_cutoff_date {
     my ($self, $collection_date) = @_;
-    return _bulky_time_object_to_datetime($collection_date, $self->bulky_cancellation_cutoff_time());
+
+    my $fn =
```

Comment:
The unit tests are presumably failing because they call these low level functions as static methods on the class, rather than instances, so $self->{c} won't/can't exist. This does feels like a too low level place to be testing the bulky/small items, I realise code is shared but something right down here shouldn't be deciding between bulky/shared, I think, we'd have to move that decision higher in the calling chain and then pass in bulky/small items, I think.

Although given my other comment, I don't think this is needed at all - I think this should, until/unless we hear otherwise, have the same cutoff as bulky, I don't see why they would differ, and this could all be ignored for now and only have the test :)